samples/secure-fraud-detection/README.md

```diff
@@ -0,0 +1,119 @@
+# Secure Fraud Detection Demo
+
+This demo showcases the implementation of a secure fraud detection system whuich is available only to users authenticated with Google.
```

Comment:
```suggestion
This demo showcases the implementation of a secure fraud detection system which is available only to users authenticated with Google.
```

Reply:
Thanks @geoand, I spotted it earlier and modified README alongside other updates locally so far, so this suggestion won't apply cleanly, let me resolve it